src/linker/runner.py

```diff
@@ -22,9 +22,10 @@ def main(
     pipeline = Pipeline(config)
 
     # Copy config files to results
```

Comment:
this looks like enough related lines to extract into a function or method IMO, i think a Config method would be neatest